package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@connectedcars/logutil",
-  "version": "2.2.0",
+  "version": "2.3.0",
```

Comment:
Wait with this until after it's merged to master. Once it's merged, you can do `npm version major|minor|patch`, which will automatically update package.json and add a git tag.